### ISSUE TITLE
Visibility setting of article not consistent

### ISSUE THREAD

**Issue by MrGeneration** [zammad/zammad#2448]
### Infos:

* Used Zammad version: 2.8
* Installation method (source, package, ..): any
* Operating system: any
* Database + version: any 
* Elasticsearch version: any
* Browser + version: Google Chrome (proberbly any)
* Ticket-ID: #1035696

### Expected behavior:

* When working with visibility a lot, the way setting an article private or public should be consistent (icons).

### Actual behavior:

* When changing the visbility of an article, the icons on existing articles and currently "to be posted articles" are different when setting article to private or active.

### Steps to reproduce the behavior:

* Add a private/public note
* act as you'd like to add another private/public note and see the differences in the icons (lock)

![image](https://user-images.githubusercontent.com/6549061/51751386-0c5b7300-20b5-11e9-9e5a-6c5c03282abd.png)

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by mrflix** [comment]
This one is tricky. Show the action or show the state?
There is no clear rule in UI design. For example in a video player the play/pause icon usually shows the action: when its playing it shows the pause icon.

![vlc playing state](https://user-images.githubusercontent.com/60777/59848283-0d8f8400-9365-11e9-97ff-e630cb691a7b.png)
![quicktime playing state](https://user-images.githubusercontent.com/60777/59848345-30219d00-9365-11e9-952a-6fda32deb8b2.png)

In Microsofts explorer the arrows show the state (open/closed):
![image](https://user-images.githubusercontent.com/60777/59848452-7840bf80-9365-11e9-9924-176352ca6288.png)

In our case the lock icon with an article currently shows the action, while the lock icon in the editor at the bottom shows the state:

![image](https://user-images.githubusercontent.com/60777/59848564-c0f87880-9365-11e9-92be-84f0b0ad0c9d.png)

"set to public" forces the icon to be the action and not the state icon. If we'd switch the lock-icon to "state mode" below the article it would just look wrong:

![image](https://user-images.githubusercontent.com/60777/60014050-5ce4f580-9680-11e9-8cea-5e859922f0ab.png)

The other way around if we'd turn the state icon at the bottom into an action icon it would look like this:

![image](https://user-images.githubusercontent.com/60777/59848831-690e4180-9366-11e9-9336-2080bca6a1cc.png)

Is it just me or does this feel wrong? Without the text next to it you expect the icon to indicate the current state, not the state after it was clicked.

In conclusion I'd reconsider staying with the current state. cc @martini @thorsteneckel

**Comment by thorsteneckel** [comment]
@MrGeneration and I had an internal discussion and we can follow your argumentation (action/state) and will keep it as it is for now - until more people complain. It was only one so far. Closing now.
